pyproject.toml

```diff
@@ -35,6 +36,7 @@ PyYAML = "*"
 dask = "*"
 toml = "*"
 ### NOT CORE
+pandas = "<=2.2.1"
```

Comment:
Since pandas is a requirement of xarray, and this is presumably done to make sure xarray installs well, I think it should go in CORE